README.md

```diff
@@ -62,6 +62,23 @@ Thank you to all of our wonderful contributors!
   <img src="https://contrib.rocks/image?repo=keras-team/keras-cv" />
 </a>
 
+
+## Installing from Source
```

Comment:
Does `python setup.py develop` still work after this change?

Reply:
Yes, `python setup.py develop` still works. Updated.